src/main/java/org/folio/rest/util/ResponseUtils.java

```diff
@@ -54,9 +53,8 @@ public static <T> Future<T> handleFailure(Throwable cause) {
   public static <T, V> void handleFailure(Promise<T> promise, AsyncResult<V> reply) {
     Throwable cause = reply.cause();
     logger.error(cause.getMessage());
-    if (cause instanceof PgException && "23F09".equals(((PgException)cause).getCode())) {
-      String message = MessageFormat.format(ErrorCodes.CONFLICT.getDescription(), ((PgException)cause).getTable(),
-        ((PgException)cause).getErrorMessage());
+    if (cause instanceof PgException pgEx && "23F09".equals(pgEx.getSqlState())) {
```

Comment:
I think string shuold be moved to static final or somewhere collect constraints

Reply:
My thinking is that static string will be memory effecient, but need to do research

Reply:
I am going to try using [PgExceptionUtil.isVersionConflict()](https://github.com/folio-org/raml-module-builder/blob/b0276549ba7be763e387f6174cca4669e46404b6/domain-models-runtime/src/main/java/org/folio/rest/persist/PgExceptionUtil.java#L81).